bugbug/github.py

```diff
@@ -79,9 +83,34 @@ def get_start_page() -> int:
     return int(count / PER_PAGE) + 1
 
 
+def fetch_issues_updated_since_timestamp(
+    owner: str, repo: str, state: str, since: str, retrieve_events: bool = False
+) -> list:
```

Comment:
This can be `list[IssueDict]`

Reply:
Getting an error with using `list`, `Class 'type' doesn't define '__getItem__', so the '[]' operator cannot be used on its instances`, so going to use `typing.List` instead of `list`